packages/graphql/src/schema/make-augmented-schema.test.ts

```diff
@@ -567,40 +567,31 @@ describe("makeAugmentedSchema", () => {
         test("should throw error if more than one @id directive field has the global argument set to true", () => {
             const typeDefs = gql`
                 type User {
-                    email: ID! @id(global: true)
-                    name: ID! @id(global: true)
+                    email: ID! @id @relayId
+                    name: ID! @id @relayId
                 }
             `;
             expect(() => makeAugmentedSchema(typeDefs)).toThrow(
-                "Only one field may be decorated with an '@id' directive with the global argument set to `true`"
-            );
-        });
-        test("should throw if an @id directive has the global argument set to true, but the unique argument set to false", () => {
-            const typeDefs = gql`
-                type User {
-                    email: ID! @id(global: true, unique: false)
-                }
-            `;
-            expect(() => makeAugmentedSchema(typeDefs)).toThrow(
-                `Fields decorated with the "@id" directive must be unique in the database. Please remove it, or consider making the field unique`
+                "Only one field may be decorated with the `@relayId` directive"
             );
         });
+
         test("should throw if a type already contains an id field", () => {
             const typeDefs = gql`
                 type User {
                     id: ID!
-                    email: ID! @id(global: true)
+                    email: ID! @id @relayId
                 }
             `;
 
             expect(() => makeAugmentedSchema(typeDefs)).toThrow(
-                `Type User already has a field "id." Either remove it, or if you need access to this property, consider using the "@alias" directive to access it via another field`
+                `Type User already has a field \`id\`, which is reserved for Relay global node identification.\nEither remove it, or if you need access to this property, consider using the \`@alias\` directive to access it via another field`
```

Comment:
I think it would be better to reduce the use of backticks in error messages. I suggest a style similar to Typescript: `"Error message on field 'id' has an error"`